po/es.po

```diff
@@ -0,0 +1,1212 @@
+"standard"
+msgstr ""
+"Puede que sigas un proceso que vaya de acuerdo con uno publicado por los estándares de "
+"ISO"
+
+#: src\iso26262.md:9
+msgid "Certification and Qualification"
+msgstr "Certificación y Calificación"
+
+#: src\iso26262.md:11
+msgid ""
+"You are responsible for **certifying** that the development of _your_ "
+"product was in accordance with such a standard."
+msgstr ""
+"Eres responsable for **certificar** que el desarrollo de _tu_ producto "
+"se hizo de acuerdo con dicho estándar."
+
+#: src\iso26262.md:13
+msgid ""
+"You might choose tools that are **qualified** as being suitable for use with "
+"such a standards compliant process."
+msgstr ""
+"Puedes escoger herramientas ya **calificadas** apropiadamente para ser usadas con "
+"un proceso que cumple un proceso estandarizado."
+
+#: src\iso26262.md:16
+msgid "What is ISO 26262?"
+msgstr "¿Qué es el ISO 26262"
+
+#: src\iso26262.md:18
+msgid ""
+"(ISO 26262-1:2018) is intended to be applied to safety-related systems that "
+"include one or more electrical and/or electronic (E/E) systems and that are "
+"installed in series production road vehicles, excluding mopeds."
+msgstr ""
+
+#: src\iso26262.md:22
+msgid ""
+"The document addresses possible hazards caused by malfunctioning behaviour "
+"of safety-related E/E systems, including interaction of these systems."
+msgstr ""
+
+#: src\iso26262.md:25
+msgid "From [iso.org](https://www.iso.org/standard/68383.html)"
+msgstr "De [iso.org](https://www.iso.org/standard/68383.html)"
+
+#: src\iso26262.md:27
+msgid "And that has qualified tools?"
+msgstr "¿Y eso tiene herramientas calificas?"
+
+#: src\iso26262.md:29
+msgid "ISO 26262-8:2018(E), Section 11.1.b says:"
+msgstr "ISO 26262-8:2018(E), Sección 11.1.b dice:"
+
+#: src\iso26262.md:31
+msgid ""
+"to provide means for the qualification of the software tool when applicable, "
+"in order to create evidence that the software tool is suitable to be used to "
+"support the activities or tasks required by the ISO 26262 series of "
+"standards (i.e. the user can rely on the correct functioning of a software "
+"tool for those activities or tasks required by the ISO 26262 series of "
+"standards)."
+msgstr ""
+
+#: src\iso26262.md:37
+msgid "Confidence in your Tools"
+msgstr "Confíar en tus herramientas"
+
+#: src\iso26262.md:39 src\iso26262.md:43
+msgid "What is the tool supposed to do?"
+msgstr "¿Qué debe hacer la herramienta?"
+
+#: src\iso26262.md:40 src\iso26262.md:49
+msgid "Does it do what it is supposed to do?"
+msgstr "¿Hace lo que se supone que tiene que hacer?"
+
+#: src\iso26262.md:41 src\iso26262.md:57
+msgid "Does someone I trust believe it does what it is supposed to do?"
+msgstr "¿Alguien en quien yo confío le cree que va a hacer lo que dice que tiene que hacer?"
+
+#: src\iso26262.md:45
+msgid "Rust doesn't have a written specification ... yet."
+msgstr "Rust todavía no tiene una especificación escrita ... aún."
```

Comment:
I would remove either the "todavia" or the "aun" to avoid the redundancy

Reply:
Cierto - voy a esperar a feedback de Pietro/Florian para refrasear esta oración debido al trabajo actual.